src/index.test.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line arrow-parens
+test('display itemscount on categories', done => {
+  function commentBtnEvent(data) {
+    try {
+      expect(data).toBe('12');
+      done();
+    } catch (error) {
+      done(error);
+    }
+  }
+  commentBtnEvent('12');
+});
```

Comment:
[OPTIONAL] Please try to disable the least amount of linter rules as possible. Remember always to search on google or eslint documentation what each rule means so you can fix it. In this case, it wants line 2 to be: test('display itemscount on categories', (done) => {`. With parenthesis enclosing the args of the arrow function.